### PR TITLE
feat: [PIE-6906]: use template call failure to show pipeline invalid text for new input set and run pipeline buttons

### PR DESCRIPTION
**Pull request by chetanvekariya-harness**
### Summary
- Do this for Input Set list page

#### Screenshots
**Before:**
Screenshot 2022-12-19 at 3 27 03 PM

**After:**
Screenshot 2022-12-19 at 3 27 07 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by chetanvekariya-harness** [comment]
retrigger cypress-pipeline

**Comment by chetanvekariya-harness** [comment]
retrigger jest

**Review by rameshx — APPROVED** [review]

File: src/modules/70-pipeline/pages/inputSet-list/InputSetList.tsx [hunk 1]
```diff
@@ -118,7 +120,12 @@ function InputSetList(): React.ReactElement {
     },
     { staleTime: 5 * 60 * 1000 }
   )
-  const isPipelineInvalid = !pipelineMetadata?.data?.entityValidityDetails?.valid
+
+  useEffect(() => {
+    if (pipelineMetadata?.data && !pipelineMetadata?.data?.entityValidityDetails?.valid) {
+      setIsPipelineInvalid(true)
+    }
+  }, [pipelineMetadata])
```
Comment (udayvunnam) [inline src/modules/70-pipeline/pages/inputSet-list/InputSetList.tsx:129]: could this be computed on render without additional effect & state? performance isn't a problem and can use useMemo if need be?

Comment (chetanvekariya-harness) [reply src/modules/70-pipeline/pages/inputSet-list/InputSetList.tsx]: Done.

**Review by rameshx — APPROVED** [review]

**Review by udayvunnam — APPROVED** [review]

**Comment by chetanvekariya-harness** [comment]
retrigger cypress-rest

**Pull request merged**